tests/perf-regression.100threads.30M-keys.yaml

```diff
@@ -12,6 +12,10 @@ space_node_threshold: 644245094
 ami_id_db_scylla_desc: 'VERSION_DESC'
 #ip_ssh_connections: 'private'
 
+# email definitions to send regression report
+send_email: true
+email_recipients: ['roy@scylladb.com', 'larisau@scylladb.com']
```

Comment:
let's merge and check the format and if it's clear.
When we agree we will add dev-int.